tests/framework/prototype.py

```diff
@@ -4,18 +4,41 @@
 import unittest
 import time
 import subprocess
+import os
 import yaml
 from kubernetes import client, config
-import docker
 import jsonpatch
 
-# Utility functions
+# Global variables
 logger = logging.getLogger(__name__)
 logging.basicConfig(
     format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
     datefmt='%Y-%m-%d:%H:%M:%S',
     level=logging.INFO)
 
+class CONST(object):
+    """Constants"""
+    __slots__ = ()
+    # Docker images
+    OPERATOR_IMAGE_KEY = "kuberay-operator-image"
+    RAY_IMAGE_KEY = "ray-image"
+
+    # Kubernetes API clients
+    K8S_CR_CLIENT_KEY = "k8s-cr-api-client"
+    K8S_V1_CLIENT_KEY = "k8s-v1-api-client"
+
+    # Paths
+    REPO_ROOT = "/" + "/".join(os.path.abspath(os.path.dirname(__file__)).split('/')[1:-2]) + "/"
+    HELM_CHART_ROOT = REPO_ROOT + "helm-chart"
```

Comment:
Use os.path.join or pathlib to form paths.

Reply:
Updated https://github.com/ray-project/kuberay/pull/759/commits/b2a23897408b00122fe5729b8190a850c63ff82b.